src/Form/Jecoute/DataAnswerFormType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace AppBundle\Form\Jecoute;
+
+use AppBundle\Entity\Jecoute\Choice;
+use AppBundle\Entity\Jecoute\DataAnswer;
+use AppBundle\Entity\Jecoute\SurveyQuestion;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class DataAnswerFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('textField', TextType::class, [
+                'required' => false,
+            ])
+            ->add('surveyQuestion', EntityType::class, [
+                'class' => SurveyQuestion::class,
+            ])
+            ->add('selectedChoices', EntityType::class, [
+                'class' => Choice::class,
+                'multiple' => true,
+            ])
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'data_class' => DataAnswer::class,
+            'csrf_protection' => false,
```

Comment:
I prefer to disable the CSRF protection in the controller (`createForm`) as this form can be used in another page with CSRF validation.